src/components/emojis.ts

```diff
@@ -0,0 +1,14 @@
+import { Emoji } from 'discord.js';
+import { SapphireClient } from '@sapphire/framework';
+
+const emojiList: { [key: string]: Emoji } = {};
+
+export const initEmojis = (client: SapphireClient): void => {
+  client.emojis.cache.forEach((emoji) => {
+    if (emoji.name) emojiList[emoji.name] = emoji;
+  });
+};
```

Comment:
huh

weird that it didn't track the file change from the oldCode folder.

Is there a way to keep the history? If not, oh well.

Reply:
I think the file contents changed too much. so it was automatically detected by git to not be a rename, rip.